taqueria-plugin-archetype/README.md

```diff
@@ -1,13 +1,12 @@
 # Archetype Plugin
 
-This is a plugin developed for Taqueria built on NodeJS using the Taqueria Node SDK and distributed via NPM
-
 The Archetype plugin provides a task to compile Archetype smart contracts (`.arl`) to Michelson `.tz` files
 
 ## Requirements
```

Comment:
If we list the latest versions of dependencies, I think it's better to use a softer language that means "recommended versions of system dependencies". The same version might still not be available on a platform the user is using.